platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/address/AddressBookDiff.java

```diff
@@ -31,9 +30,9 @@
  * @param removedNodes           the nodes that were removed
  * @param modifiedNodes          the nodes that were modified
  */
-public record RosterDiff(
-        @NonNull UpdatedRoster newRoster,
-        boolean rosterIsIdentical,
+public record AddressBookDiff(
```

Comment:
Neither the `RosterDiff` or the `RosterDiffGenerator` are in use in the rest of the code.  This was added under the old paradigm of doing DynamicAddressBook phase 3.  That's completely out the window in terms of design.  I think this code should be outright deleted, but I will defer to @litt3's opinion

Reply:
removed